.changeset/two-years-pretend.md

```diff
@@ -0,0 +1,6 @@
+---
+'@leafygreen-ui/search-input': major
+'@leafygreen-ui/text-input': patch
+---
+
+Adds console warning suggestion to use SearchInput component in place of TextInput with type="search"
```

Comment:
Maybe break these out into separate files? So we can have different messages. The patch with this comment, and a second one that releases the search-input